lists/global.csv

```diff
@@ -1234,3 +1234,10 @@ https://pt.m.wikipedia.org/,CULTR,Culture,2019-02-19,Wikimedia,URLs provided by
 https://mitm.watch/,ANON,Anonymization and circumvention tools,2019-03-20,magma,This site detects encryption-removing MITM using logic from NDSS17 interception paper
 https://credibilitycoalition.org/,CULTR,Culture,2019-04-19,OONI,Reportedly inaccessible on WE in Egypt (but probably worth testing globally)
 https://black.riseup.net/,ANON,Anonymization and circumvention tools,2019-04-26,LEAP,
+https://gitlab.com/,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
+https://termux.com/,HOST,Hosting and Blogging Platforms,2019-07-17,chelchela,
```

Comment:
Is there a specific reason to test the homepage of this software?

Reply:
At the first start of the Termux app, it must communicate with its site, otherwise the application will remain in loading mode.

I made a mistake
The termux.org and termux.net addresses are used :
https://github.com/termux/termux-app/blob/370ac2bd8911b5632933975300e779f722ad03af/app/src/main/java/com/termux/app/TermuxInstaller.java#L174-L175

The following URLs should be tested for proper analysis:
https://termux.org/bootstrap-arm.zip
https://termux.net/bootstrap/bootstrap-arm.zip